cartographer/io/image_utils.h

```diff
@@ -0,0 +1,27 @@
+#ifndef CARTOGRAPHER_IO_IMAGE_UTILS_H_
+#define CARTOGRAPHER_IO_IMAGE_UTILS_H_
+
+#include <cstdint>
+#include <vector>
+
+namespace cartographer {
+namespace io {
+
+class FileWriter;
```

Comment:
Ok. I see that you seem to repeat includes from the header in the source files. Should I do the same?